<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BatchCreator_MainWindow</class>
 <widget class="QMainWindow" name="BatchCreator_MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1053</width>
    <height>788</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1C1C1C;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="left_vertical_frame">
      <property name="maximumSize">
       <size>
        <width>340</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>330</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTreeView" name="MiniWindows_explorer">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="horizontal_frame">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>32</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTextEdit" name="Status_Line_Qedit"/>
          </item>
          <item>
           <widget class="QToolButton" name="Copy_from_status_line_toolButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/file_copy_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/file_copy_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="file_initialize_button">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/add_circle_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/add_circle_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="status_label">
         <property name="text">
          <string>Oppened File: BatchCreator version:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="folder_path_template">
            <property name="styleSheet">
             <string notr="true">
    /* QPushButton default and hover styles */
    QLabel {

        font: 600 10pt &quot;Segoe UI&quot;;
	

        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        padding-top: 4px;
        padding-bottom:4px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QLabel:hover {
        background-color: #414956;
        color: white;
    }</string>
            </property>
            <property name="text">
             <string>Folder path</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="assets_name_template">
            <property name="styleSheet">
             <string notr="true">
    /* QPushButton default and hover styles */
    QLabel {

        font: 600 10pt &quot;Segoe UI&quot;;
	

        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        padding-top: 4px;
        padding-bottom:4px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QLabel:hover {
        background-color: #414956;
        color: white;
    }</string>
            </property>
            <property name="text">
             <string>Asset name</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="kv3_QplainTextEdit">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="plainText">
          <string/>
         </property>
         <property name="placeholderText">
          <string>Content</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="extension_lineEdit">
         <property name="placeholderText">
          <string>File extension (vmdl, vmat etc).</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="open_button">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>18</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
    /* QPushButton default and hover styles */
    QPushButton {

        font: 600 10pt &quot;Segoe UI&quot;;
	

        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:22px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QPushButton:hover {
        background-color: #414956;
        color: white;
    }
    QPushButton:pressed {
        background-color: red;
        background-color: #1C1C1C;
        margin: 1 px;
        margin-left: 2px;
        margin-right: 2px;

    }</string>
            </property>
            <property name="text">
             <string>Open</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/file_open_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</normaloff>:/icons/file_open_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="save_button">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>18</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
    /* QPushButton default and hover styles */
    QPushButton {

        font: 600 10pt &quot;Segoe UI&quot;;
	

        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:22px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QPushButton:hover {
        background-color: #414956;
        color: white;
    }
    QPushButton:pressed {
        background-color: red;
        background-color: #1C1C1C;
        margin: 1 px;
        margin-left: 2px;
        margin-right: 2px;

    }</string>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/save_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</normaloff>:/icons/save_16dp_9D9D9D_FILL0_wght400_GRAD0_opsz20.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="process_all_button">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>18</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">
    /* QPushButton default and hover styles */
    QPushButton {

        font: 600 10pt &quot;Segoe UI&quot;;
	

        border: 2px solid black;
        border-radius: 4px;
        border-color: rgba(80, 80, 80, 255);
        height:22px;
        padding-top: 2px;
        padding-bottom:2px;
        padding-left: 4px;
        padding-right: 4px;
        color: #E3E3E3;
        background-color: #1C1C1C;
    }
    QPushButton:hover {
        background-color: #414956;
        color: white;
    }
    QPushButton:pressed {
        background-color: red;
        background-color: #1C1C1C;
        margin: 1 px;
        margin-left: 2px;
        margin-right: 2px;

    }</string>
            </property>
            <property name="text">
             <string>Process all</string>
            </property>
            <property name="icon">
             <iconset resource="../resources.qrc">
              <normaloff>:/icons/tab_move_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</normaloff>:/icons/tab_move_24dp_9D9D9D_FILL0_wght400_GRAD0_opsz24.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
